server/repository/src/db_diesel/changelog/test.rs

```diff
@@ -0,0 +1,572 @@
+use super::changelog::changelog::dsl as changelog_dsl;
+use diesel::prelude::*;
+use util::{inline_edit, inline_init};
+
+use crate::{
+    mock::{
+        mock_item_a, mock_name_a, mock_name_b, mock_name_store_a, mock_name_store_b, MockData,
+        MockDataInserts,
+    },
+    test_db::{self, setup_all, setup_all_with_data},
+    ChangelogAction, ChangelogFilter, ChangelogRepository, ChangelogRow, ChangelogTableName,
+    EqualFilter, InvoiceLineRow, InvoiceLineRowRepository, InvoiceRow, InvoiceRowRepository,
+    NameRow, NameRowRepository, RequisitionLineRow, RequisitionLineRowRepository, RequisitionRow,
+    RequisitionRowRepository, StorageConnection, StoreRow,
+};
+
+#[actix_rt::test]
+async fn test_changelog() {
```

Comment:
Relocated existing test (and used new interface for getting changelogs), but added new test below, see comment